tests/gdb-tests/tests.py

```diff
@@ -176,6 +182,14 @@ def parse_args():
     if args.pdb:
         print("Will run tests in serial and with Python debugger")
         args.serial = True
+    if args.nix:
+        gdbinit_path = os.path.join(ROOT_DIR, "result/share/pwndbg/gdbinit.py")
+        if not os.path.exists(gdbinit_path):
+            print("ERROR: No nix-compatible gdbinit.py found. Run nix build .#pwndbg-dev")
+            sys.exit(1)
+        GDB_INIT_PATH = os.path.join(ROOT_DIR, "result/share/pwndbg/gdbinit.py")
+        # This tells tests/utils.py where to find the gdbinit.py file when used by various tests
+        os.environ["GDB_INIT_PATH"] = GDB_INIT_PATH
```

Comment:
Why not just use `gdbinit_path` here? I don't think we need to define the `GDB_INIT_PATH` in the previous line.